src/app/core/facets-manager/facets-manager.component.ts

```diff
@@ -276,6 +294,64 @@ export class FacetsManagerComponent implements OnInit, OnDestroy, AfterViewInit
           }
 
         });
+        } 
```

Comment:
My IDE is yelling at me about a bunch of tsLint issues in this file. Do you have the display of those turned off? I'd like to keep everything as clean as possible, at least for the /core files where other orgs may want to work with them. I can clean this up after the merge request but it's a good thing to keep in mind for the future